.ci_support/linux_64_r_base4.3.yaml

```diff
@@ -22,8 +22,6 @@ r_base:
 - '4.3'
 target_platform:
 - linux-64
-tiledb:
-- '2.26'
```

Comment:
Not sure what changed to remove these pins. They are unnecessary since we pin the version in the recipe, which causes these values to be ignored anyways. But I didn't see any recent update to conda-smithy that indicated this improved behavior

Reply:
We just saw this in tiledb-py-feedstock too, was about to ping you to see if you knew why!

Reply:
It is strange why it is only just happening now. I don't see any recent update to conda-smithy/conda-build/conda that would be related to this. And tiledb is [still pinned in conda-forge-pinning](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/e6c41e91ba8a8f4f2edb087f5e8be6428c6c3374/recipe/conda_build_config.yaml#L893). As long as recipes that depend on the global tiledb pin continue to be pinned, we don't need to worry about this. In other words, we'll need to monitor the tiledb 2.27 migration to make sure it proceeds properly.

Reply:
I think the changing behavior is due to the version of conda-build. Basically the pins are added by conda-build 24.9.0 but not by 24.5.1 (and earlier)

* The most recent migration PR was for tiledb 2.25 back in July (#85). It used conda-build 24.5.1, conda-smithy 3.37.1, and conda-forge-pinning 2024.07.25.21.45.50. I didn't realize it at the time since I closed it, but it didn't actually update any pins in `.ci_support/`. This indicates that this behavior of noticing that tiledb was pinned in the recipe already existed for months
* There was no migration PR for tiledb 2.26, suggesting that the migrators have updated behavior
* But then just last week the bot rerendering for the 0.30.2 update PR (#90) added new pins for tiledb 2.26. It used conda-build 24.9.0, conda-smithy 3.43.0, and conda-forge-pinning 2024.10.24.08.44.27
* Then this PR removed the pins again. I used conda-build 24.5.1, conda-smithy 3.43.1, and conda-forge-pinning 2024.10.31.07.03.28.

Looking at past migration PRs for the tiledb pin, at least in this feedstock, they haven't actually modified the pins in `.ci_support/` in a while